client/state/sites/intro-offers/reducers.ts

```diff
@@ -0,0 +1,61 @@
+import { Dictionary, keyBy } from 'lodash';
```

Comment:
Not a blocker, but I know the calypso team is trying to remove instances of lodash (there's even [a whole package](https://github.com/Automattic/wp-calypso/tree/trunk/packages/js-utils) devoted to it) and this PR adds a few. Is it possible to do these things without lodash and/or could we add functions to that package to avoid having to import lodash?

Reply:
Just curious about that decision...

Has the team considered a better alternative to lodash like [Ramda](https://ramdajs.com/)?
It's ultra lightweight and is designed for functional programming and makes creating and using predicates a breeze.

Reply:
@manzoorwanijk You can find the original context for the initiative here: `p4TIVU-9Bf-p2` And there's always MGS if you want to dig deeper 😉

Reply:
Ah Thanks